test/unit/model/v2/test_platforms_section.py

```diff
@@ -164,6 +166,7 @@ def _model_platforms_docker_errors_section_data():
             },
             'override_command': int(),
             'command': int(),
+            'tty': str(),
```

Comment:
Maybe `bool()`?

Reply:
```suggestion
            'tty': bool(),
```

Reply:
I do not agree. This is inside `_model_platforms_docker_errors_section_data()` which is supposed to generate invalid data on purpose in order to test the model schema - unless I somehow misunderstood everything there.

Reply:
Oh, alright.